core/src/test/scala/integration/kafka/admin/ReassignPartitionsIntegrationTest.scala

```diff
@@ -128,11 +126,10 @@ class ReassignPartitionsIntegrationTest extends ZooKeeperTestHarness {
         PartitionReassignmentState(Seq(3, 2, 0), Seq(3, 2, 0), true)
     )
 
-    // When using --zookeeper, we aren't able to see the new-style assignment
```

Comment:
Since we are removing this comment, should we be using new style assignment?

Reply:
This comment is kind of weird here, but as [here](https://github.com/apache/kafka/pull/10471/files#diff-7282fb89afcc2ee0a045cc7f5c182691a9279f85d8a9c178471253c13953f6bdL366) mentioned:

> Warning: because you are using the deprecated --zookeeper option, the results may be incomplete.  Use --bootstrap-server instead for more accurate results.

So, I add one more verification here due to we change to adminClient now.
Thank you.

Reply:
Yeah, I don't know what it means by new-style assignment either.